certified-connectors/InfoQuery Title Explorer/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4061 @@
+                      "FileID": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "FileID"
+                      },
+                      "Expedite": {
+                        "type": "boolean",
+                        "description": "Expedite"
+                      }
+                    }
+                  },
+                  "description": "Notes"
+                }
+              }
+            }
+          }
+        },
+        "description": "Gets a list of the notes on the specified file.",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "FileID",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "File identification number.",
+            "x-ms-summary": "FileID",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/GetNote": {
+      "get": {
+        "operationId": "GetNote",
+        "summary": "Get Note",
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "NoteID": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "NoteID"
+                },
+                "Subject": {
+                  "type": "string",
+                  "description": "Subject"
+                },
+                "Body": {
+                  "type": "string",
+                  "description": "Body"
+                },
+                "CreatedBy": {
+                  "type": "object",
+                  "properties": {
+                    "UserID": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "UserID"
+                    }
+                  },
+                  "description": "CreatedBy"
+                },
+                "CreatedDate": {
+                  "type": "string",
+                  "description": "CreatedDate"
+                },
+                "FileID": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "FileID"
+                }
+              }
+            }
+          }
+        },
+        "description": "Gets the note with the specified ID.",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "NoteID",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "NoteID",
+            "description": "Note identification number.",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/AddActionToFile": {
+      "post": {
+        "operationId": "AddActionToFile",
+        "summary": "Add Action To File",
```

Comment:
@tarynsnell please make sure that everything is grammatically correct and is free of typographical errors. You can refer to [English strings recommendations](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) for more information.